exasol/ds/sandbox/cli/commands/setup_vm_bucket_waf.py

```diff
@@ -7,20 +7,17 @@
 from exasol.ds.sandbox.lib.aws_access.aws_access import AwsAccess
 from exasol.ds.sandbox.lib.config import default_config_object
 from exasol.ds.sandbox.lib.logging import set_log_level
-from exasol.ds.sandbox.lib.vm_bucket.vm_dss_bucket_waf import run_setup_vm_bucket_waf
+from exasol.ds.sandbox.lib.s3.waf import Waf
 
 
 @cli.command()
 @add_options(aws_options)
 @add_options(logging_options)
 @click.option('--allowed-ip', type=str,
               help="The allowed IP address for which CAPTCHA will not be applied.")
-def setup_vm_bucket_waf(
-            aws_profile: str,
-            allowed_ip: str,
-            log_level: str):
+def setup_vm_bucket_waf(aws_profile: str, allowed_ip: str, log_level: str):
     """
     Command to deploy the VM S3-Bucket Web Application Firewall. Needs to run before deploying the VM Bucket itself.
     """
     set_log_level(log_level)
-    run_setup_vm_bucket_waf(AwsAccess(aws_profile), allowed_ip, default_config_object)
+    Waf.vm_bucket(AwsAccess(aws_profile), default_config_object).setup(allowed_ip)
```

Comment:
Be careful and don't put too much logic in the command functions. As far as I remember those functions are not covered by the unit tests....

Reply:
Yes, you are right. And yes I agree.
Would it help to just hand the value of argument `purpose` to a method in the lib?

Reply:
@tkilias proposed to move the static methods to separate classes or files:
```python
vm_bucket = VmBucket.S3Bucket(...)
vm_waf = VmBucket.Waf(...)
```

Reply:
But then class VmBucket needs to access or provide or contain the specifications for both the bucket and the waf, which results in a more spaghetti code.

Reply:
see next push